Docs/developer_guide/module_overview.md

```diff
@@ -5,22 +5,23 @@ Slicer supports multiple types of modules:
 * [Loadable](#loadable-modules)
 * [Scripted](#scripted-modules)
 
-While the developer has to choose between one of the types to implement its module, the end user will NOT notice the difference as they all share the same look & feel.
+While the developer has to choose between one of the types to implement its module, the end user will **NOT** notice the difference as they all share the same look & feel.
```

Comment:
> as they all share the same look & feel.

I would argue that CLI modules have a different look & feel compared to Loadable and Scripted.

Reply:
Since we now have the rational highlighted (See below), what about just removing this sentence ?


|  | Rational |
|--|--|
| CLI | ![image](https://user-images.githubusercontent.com/219043/202318205-0f36d1ac-1b5b-49b3-bd86-16e71e37aed7.png) |
| Loadable| ![image](https://user-images.githubusercontent.com/219043/202318353-ba95e968-fd96-456d-a525-9c01f4c7209f.png) |
| Scripted | ![image](https://user-images.githubusercontent.com/219043/202318390-da7eadae-df85-4de9-acfd-e0bdcaf48f68.png) |

Reply:
And your comment makes sense, especially considering the purpose of implementing a `loadable` module (either C++ or scripted) is to have custom and tailored user experience that can not be obtained using the automatically generated UI.

Reply:
Yes I think this sentence can be removed now that there are these rationales about when to use whichever one